apps/docs/docs/ref/kotlin/installing.mdx

```diff
@@ -0,0 +1,267 @@
+---
+id: installing
+title: 'Installing & Initialization'
+slug: installing
+custom_edit_url: https://github.com/supabase/supabase/edit/master/web/spec/supabase.yml

```

Comment:
@gregnr Should we leave this one out? Probably makes little sense to link to the `supabase.yml` other `installing.mdx` do this too though

Reply:
TBH, I'm not sure if this is actually used anywhere at the moment. Let's track in another issue: https://github.com/supabase/supabase/issues/14272